modules/twinklespeedy.js

```diff
@@ -2087,6 +2109,8 @@ Twinkle.speedy.callback.evaluateUser = function twinklespeedyCallbackEvaluateUse
 		});
 	}
 
+	var requestsalt = form.salting.checked || false;
```

Comment:
```suggestion
	var requestsalt = form.salting.checked;
```
and there's no need for this variable as it's used just once.

Reply:
Removed variable